src/components/Card/Card.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import './Card.css';
+import type { Country }  from '../../models/Country';
+
+interface ICountry {
+    country: Country
+}
+
+const Card = ({ country }: ICountry) => {
+    return (
+        <div className="card">
+            <img src={country.image} alt="" />
```

Comment:
Could you put the name in the alt of each image?
`<img src={country.image} alt={country.name} />`

Reply:
You are right! Thanks!

Reply:
You could use tag Image from React

Reply:
Also you could standar de Types for react components like this

`const Card: FC<ICountry> = ({ country }) => {`

Reply:
For the last advice, the linter show me that is
`const Card: React.FC<ICountry> = ({ country }) = > { }`

Reply:
@carloshs92 "Image from React" is it not for ReactNative?

Reply:
I realize that Image is from NextJS also but not for react xD